File: Hackerrank/Algorithms/Implementation/Viral Advertising.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */  
    int n;
    cin >> n;
    long long sum = 0;
    long long a = 2;
    n--;
    
    while(n>=0)
    {
       sum += a;
       a = (a*3)/2;
       n--;
    }
    
    cout << sum;
    return 0;
}

File: Hackerrank/Algorithms/Implementation/Viral Advertising.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int s = 0;
        int start = 5;
        
        while (n>0) {
            s += start/2;
            start = (start/2)*3;
            n -= 1;
        }
        System.out.println(s);
    }
}